src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/DragAndDropTests/DragDrop_ListViewReorder_Automated.cs

```diff
@@ -1,5 +1,4 @@
-﻿#if false
```

Comment:
Apparently I need to look more closely when I check whether a PR has tests associated :p